test/test.js

```diff
@@ -2864,6 +2864,16 @@ exports['wrap EventEmitter (or jQuery) on handler'] = function (test) {
     });
 };
 
+exports['removing EventEmitter (or jQuery) listener on destruction'] = function(test) {
```

Comment:
Can you make a note of the original issue? Just append " (issue #500)" to the end.